### PR TITLE
[SITES-554] Add accessibility tests and fix accessibility errors

### PR DESCRIPTION
**Pull request by tessereth**
Notes:
* Running `rspec` will test accessibility of a couple of dummy pages
* Running `rake sitemap:create` will create a sitemap of the public site in `public/sitemaps/public.xml` and a dummy editorial sitemap in `public/sitemaps/editorial.xml` as well as the index `public/sitemap.xml`
* Running `ACCESSIBILITY_TEST_SITEMAP='public/sitemaps/public.xml' rake smoke:accessibility` will test accessibility of all pages listed in the sitemap.
* Fixes accessibility issues with the news tags and alt-text for minister images.

### CODE REVIEW

**Comment by freshtonic** [comment]
Looks flippin' awesome, @tessereth 👍

LGTM

File: spec/features/accessibility_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  WebMock.allow_net_connect!
+
+  shared_examples 'is accessible' do |url, as_code|
+    it "#{url} is accessible" do
+      if as_code
+        url = eval(url)
+      end
+      visit url
+      expect(page.status_code).to eq(200)
+      # TODO: Remove skip clause once ui-kit colours are updated
```
Comment (maxious) [inline spec/features/accessibility_spec.rb]: please raise a bug on ui-kit.

**Comment by maxious** [comment]
Having to do `brew install qt` but LGTM

**Comment by bjgordon** [comment]
Same issue as @maxious. Add the requirement for `brew install qt` to README if this is the way we go.

**Pull request merged**